knowledge-base/actions/svenstaro/action-security.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Upload files to a GitHub release'
```

Comment:
@vandana41 please correct the `file-path`, the correct one would be `knowledge-base/actions/svenstaro/upload-release-action/action-security.yml`